enigma-js/src/Enigma.js

```diff
@@ -496,6 +496,44 @@ export default class Enigma {
     return task;
   }
 
+  /**
+   * Verify ENG network output matches output registered on ETH
+   *
+   * @param {Task} task - Task wrapper for contract deployment and compute tasks
+   * @return {boolean} True/false on whether outputs match
+   */
+  async verifyTaskOutput(task) {
+    const ethOutputHash = await this.getTaskOutputHash(task);
+    const engOutputHash = this.web3.utils.soliditySha3(
+      {t: 'bytes', value: task.encryptedAbiEncodedOutputs.toString('hex')}
+    );
+    return ethOutputHash === engOutputHash;
+  }
+
+  /**
+   * Verify ENG network status matches status registered on ETH
+   *
+   * @param {Task} task - Task wrapper for contract deployment and compute tasks
+   * @return {boolean} True/false on whether statuses match
+   */
+  async verifyTaskStatus(task) {
+    const ethStatus = (await this.getTaskRecordStatus(task)).ethStatus;
+    switch (task.engStatus) {
+      case 'SUCCESS':
+        return ethStatus === 2;
```

Comment:
Could you please use constants instead?

Reply:
Implemented in dcde0fb, so much better, thanks for the suggestion.